### ISSUE TITLE
Guide SMT - incorporer dans l'IG documentation ?

### ISSUE THREAD

**Issue by nriss** [ansforge/IG-documentation#42]
https://industriels.esante.gouv.fr/sites/default/files/media/document/Global_features_FHIR_Server_version_finale_v4.pdf

**Comment by 3abdel3ali** [comment]
@nriss YES ! désolé de répondre que maintenant ! voici la dernière version: https://industriels.esante.gouv.fr/sites/default/files/media/document/Global_features_FHIR_Server_version_finale_v5.pdf
